README.md

```diff
@@ -12,6 +12,13 @@ Check the [documentation](https://scholarly.readthedocs.io/en/latest/?badge=late
 
 ## Installation
 
+### To install modified version here:
+```bash
+pip3 install -U git+https://github.com/Jianbin-IBM/scholarly.git

```

Comment:
We will keep the current repository. We are not going to put pointers to derivative repositories.